package.json

```diff
@@ -31,14 +31,14 @@
     "external-services": "docker compose --file packages/hash/external-services/docker-compose.yml --env-file .env",
     "fix": "npm-run-all --continue-on-error \"fix:*\"",
     "@todo.1": "Remove the ignore argument below for mock-block-dock once repo is upgraded fully to 0.2 spec",
-    "fix:dependency-version-consistency": "check-dependency-version-consistency --fix . --ignore-dep=mock-block-dock",
+    "fix:dependency-version-consistency": "check-dependency-version-consistency --fix . --ignore-dep=mock-block-dock --ignore-dep=@blockprotocol/graph --ignore-dep=block-scripts",
```

Comment:
Any thoughts on how we can track this? 

Really wish we could put TODO comments in package.json 😭

Reply:
We can (sort of), there's one two lines above this 

`yarn` will warn you if one of these checks is _unused_, i.e. we will be immediately told if we've made everything consistent and this remains... but if inconsistencies remain then we just need to somehow externally remember that they need fixing (maybe a clean up task for 0.3, but might not be a priority to go through all the other packages and give them the new versions)